core/controllers/feedback.py

```diff
@@ -282,3 +282,29 @@ def post(self):
         email_manager.send_suggestion_email(
             exploration.title, exploration.id, suggestion.author_id,
             exploration_rights.owner_ids)
+
+
+class InstantFeedbackEmailHandler(base.BaseHandler):
+    """Handles task of sending feedback message emails instantsly."""
+
+    def post(self):
+        payload = json.loads(self.request.body)
+        user_id = payload['user_id']
+        reference = payload['reference']
```

Comment:
For consistency, why not use payload['exploration_id'] etc. directly, as in SuggestionHandler?